html5/render/browser/extend/api/dom.js

```diff
@@ -55,6 +90,9 @@ const meta = {
     name: 'scrollToElement',
     args: ['string', 'object']
   }, {
+    name: 'getComponentRect',
+    args: ['string', 'string']
```

Comment:
['string', 'function'] -> callbackId is a special case for register type system.